core/domain/user_services.py

```diff
@@ -41,6 +41,8 @@
+# The system usernames are reserved usernames, before adding new value to this
+# dict make sure that there isn't any similar usernames in the datastore.
```

Comment:
Add comment here too: All bot user IDs and usernames should start with "Oppia" and end with "Bot".

Reply:
Done!